docs/INSTALLATION.md

```diff
@@ -4,11 +4,22 @@
 
 Behind the scenes, this script downloads a binary from the latest GitHub release.
 
-**This will activate `sudo` to write the executable.** Please review the install script before
-running it.
+### Windows
+
+Run this command on PowerShell:
+```shell
+irm https://raw.githubusercontent.com/spenserblack/gengo/main/install.ps1 | iex
+```
+If this doesn't work, try to run following command on PowerShell with administrative privileges and then rerun script:
+```shell
+Set-ExecutionPolicy Unrestricted
+```
```

Comment:
Instead of telling the users what execution policy they should set, I'd rather  encourage them to make an educated decision.

Thankfully, execution policy errors are pretty informative IIRC, so I think it's safe to just encourage the user to read the error message and determine the execution policy that is appropriate for them.

Something like

> If you get an error regarding execution policy, please read the error carefully and determine the execution policy that is right for you. You may try re-running the installation script if you have updated the execution policy.